app/grants/models/grant.py

```diff
@@ -16,13 +16,141 @@
 
 from .subscription import Subscription
 from .contribution import Contribution
-from .grant_clr import GrantCLR
+from .grant_clr_calculation import GrantCLRCalculation
+from .grant_collection import GrantCollection
 
 from grants.utils import get_upload_filename, is_grant_team_member
 
 from townsquare.models import Favorite
 from web3 import Web3
 
+class GrantCLR(SuperModel):
```

Comment:
Why are we not keeping this in it's own file?

Reply:
@thelostone-mc Grant and GrantCLR are coupled and having GrantCLR in its own file was causing a circular dependency. Grant directly references GrantCLR in its `calc_clr_round` method (line 509), and vice versa GrantCLR references Grant in its `grants` method (line 128). I felt like keeping GrantCLR with Grant was the quickest way to move forward, but I'm happy to work on separating them.

Reply:
Ah gotcha !